airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/source.py

```diff
@@ -6,17 +6,20 @@
 
 from airbyte_cdk.sources import AbstractSource
 from airbyte_cdk.sources.streams import Stream
-from airbyte_cdk.sources.streams.http.auth import Oauth2Authenticator
+from airbyte_cdk.sources.streams.http.auth import Oauth2Authenticator, BasicHttpAuthenticator
 
 from .streams import Applications, Interviews, Notes, Offers, Opportunities, Referrals, Users
 
 
 def _auth_from_config(config):
-    return Oauth2Authenticator(
-        client_id=config["credentials"]["client_id"],
-        client_secret=config["credentials"]["client_secret"],
-        refresh_token=config["credentials"]["refresh_token"],
-        token_refresh_endpoint=f"{SourceLeverHiring.URL_MAP_ACCORDING_ENVIRONMENT[config['environment']]['login']}oauth/token",
+    if config and config["credentials"] and 'api_key' in config["credentials"]:
```

Comment:
can you add another checking on the  auth_type? when the auth_type is `"Api Key"`, BasicHttpAuthenticator will be used.

Reply:
done, let me know if that's alright for you